testsuite/python/save_checkpoint.py

```diff
@@ -325,8 +325,14 @@
 checkpoint.register("particle_force0")
 checkpoint.register("particle_force1")
 if espressomd.has_features("COLLISION_DETECTION"):
-    system.collision_detection.set_params(
-        mode="bind_centers", distance=0.11, bond_centers=harmonic_bond)
+    if espressomd.has_features("VIRTUAL_SITES_RELATIVE"):
+        protocol = espressomd.collision_detection.BindAtPointOfCollision(
+            distance=0.11, bond_centers=harmonic_bond,
```

Comment:
On second thought, we could give each protocol a different distance, since they are supposed to be completely independent from one another. This is actually what we do with most other features that provide multiple modes.